gateways/checkoutdotcom/request_builder.go

```diff
@@ -0,0 +1,59 @@
+package checkoutdotcom
+
+import (
+	"github.com/BoltApp/sleet"
+	"github.com/checkout/checkout-sdk-go/common"
+	"github.com/checkout/checkout-sdk-go/payments"
+)
+
+func buildChargeParams(authRequest *sleet.AuthorizationRequest) *payments.Request {
+	var source = payments.CardSource{
+		Type: "card",
+		Number: authRequest.CreditCard.Number,
```

Comment:
authRequest wouldn't generally have the cc number. All our cc numbers are stored within tokenizer.

Reply:
Assuming we follow existing pattern, the CC number will end up being a string that tokenizer recognizes.

This will get substituted into the request by tokenizer after the request gets proxied through it